cli/Cargo.toml

```diff
@@ -3,6 +3,7 @@ authors = ["Dustin J. Mitchell <dustin@mozilla.com>"]
 edition = "2018"
 name = "taskchampion-cli"
 version = "0.3.0"
+publish = false
```

Comment:
It is intentional.  I don't think the cli should be published as a crate, as its API surface is the CLI, not Rust.  We can always change that, of course..

Reply:
I reckon it makes sense to publish the CLI as a crate so it can be installed via `cargo install ...`

There's quite a lot of Rust-developed CLI tools published that way (e.g I currently have ripgrep, gitui, mdbook, sccache installed this way, along with a bunch of other specifically Rust-oriented tools like `cargo-*` things). The CLI should definitely be made available via other means, but for the minimal effort required it seems worthwhile, and without many downsides

I think some distros like Debian also find it easier if tools are available on crates.io - e.g https://salsa.debian.org/rust-team/debcargo-conf/blob/master/README.rst describes pulls package sources from crates.io

Reply:
Fair enough.  I'll remove that line and ship the rest.